virttest/qemu_vm.py

```diff
@@ -2400,21 +2400,6 @@ def sort_key(dev):
                 if ats:
                     add_virtio_option("ats", ats, devices, device, dev_type)
 
-        # Add extra root_port at the end of the command line only if there is
-        # free slot on pci.0, discarding them otherwise
-        pcie_extra_root_port = int(params.get('pcie_extra_root_port', 0))
```

Comment:
We have this section for supporting device hotplugging scenarios so it couldn't be deleted.

Reply:
Ok, drop the commit to keep original code.